File: lua/refactoring/tests/refactor/106/cpp/prompt_func_param/extract.start.cpp
#include <stdio.h>

int simple_function(int a) {
  int test = 1;
  int test_other = 1;

  for (int i = 0; i < test_other; i++) {
    printf("%d %d", a, test_other);
  }

  return test;
}

File: lua/refactoring/tests/refactor/106/csharp/prompt_func_param_no_return/extract.start.cs
namespace Test
{
    class Extract
    {
        public static void simpleFunction(int a) {
            int test = 1;
            int test_other = 1;

            for (int i = 0; i < test_other; i++) {
                Console.WriteLine("{0} {1}", a, test_other);
            }
        }
    }
}

